pkg/controller/job/job_controller.go

```diff
@@ -49,6 +49,7 @@ import (
 )
 
 const statusUpdateRetries = 3
+const maxJobBackOff = 360 * 1000000000
```

Comment:
what motivates the value of `360` for base?
ideally the constant should be moved under the function.

Reply:
I put the const here so that it is close to the MaxJobBackOff var